addons/binding/org.openhab.binding.network/README.md

```diff
@@ -1,7 +1,7 @@
 # Network Binding
 
 This binding allows to check, whether a device is currently available on the network.
-This happens by either using [ping](https://en.wikipedia.org/wiki/Ping_%28networking_utility%29) or by a successful TCP connection on a port of your choosing.
+This happens by either using [ping](https://en.wikipedia.org/wiki/Ping_%28networking_utility%29), [arping](https://en.wikipedia.org/wiki/Arping) or by a successful TCP connection on a port of your choosing.
```

Comment:
Unfortunately I can't tell much about the difference between the different ping methods.